modules/ROOT/pages/security/subscriptions-authorization.adoc

```diff
@@ -0,0 +1,64 @@
+[[subscriptions-authorization]]
+:description: This page describes how to set up authorization features for subscriptions in the Neo4j GraphQL Library.
+= Subscriptions authorization
+
+Subscriptions require their own authorization rules, which are configured with the `@subscriptionsAuthorization` directive. These rules are different to normal authorization rules because only filtering rules are available for subscriptions events and subscriptions events have more limitations in how they can be filtered.
```

Comment:
```suggestion
Subscriptions require their own authorization rules, which are configured with the `@subscriptionsAuthorization` directive.
These rules are different to normal authorization rules because only filtering rules are available for subscriptions events and subscriptions events have more limitations in how they can be filtered.
```
(line break)